examples/pthreads/CIVL-0.17_2046/27_Boop_simple_vf_false.c

```diff
@@ -0,0 +1,83 @@
+#include <pthread.h>
```

Comment:
Let's talk about how to organize these when we meet to review the meat of pthreads pull request.

I'm not sure that all of these examples belong in our regression tests - I'll look into these examples more before we meet, so I know what is in them, what works, what doesn't, etc.